enhancements/two-node-fencing/tnf.md

```diff
@@ -516,19 +556,33 @@ without keepalived at all by using the ipaddr2 resource agent for pacemaker to r
-we were advised against using the Bare Metal Operator as an inventory. Its purpose/scope is provisioning nodes.
-
-This means that the Baremetal Operator is not initially in scope for a two-node cluster because we don't intend to support compute nodes. However, if this requirement were to change for future
-business opportunities, it may still be useful to provide the user with an install-time option for deploying the Baremetal Operator.
+`platform: baremetal` BMC entries would be loaded into `BareMetalHost` CRDs and we'd extend BMO to initialize pacemaker. After a discussion with the Bare Metal Platform team, we were advised against
+using the Bare Metal Operator because the capabilities of fencing in BMO depend on a functioning read-write control plane. See [the explanation
+above](#why-dont-we-reuse-the-existing-apis-in-the-baremetal-platform) for other concerns about reusing this API.
 
 Given the likelihood of customers wanting flexibility over the footprint and capabilities of the platform operators running on the cluster, the safest path forward is to target TNF clusters on both
 `platform: none` and platform `platform: baremetal` clusters.
 
-For `platform: none` clusters, this will require customers to provide an ingress load balancer. That said, if in-cluster networking becomes a feature customers request for `platform: none` we can work
-with the Metal Networking team to prioritize this as a feature for this platform in the future.
+For `platform: none` clusters, this will require customers to provide a load balancer for ingress and API traffic. That said, if in-cluster networking becomes a feature customers request for
+`platform: none` we can work with the Metal Networking team to prioritize this as a feature for this platform in the future. Some discussion for this has already begun in this [enhancement
+proposal](https://github.com/openshift/enhancements/pull/1666).
+
+#### Handling Failures via RHEL HA
+In this section we explore in detail the steps taken to recover a cluster in the event of a node or network-level failure. The bottom line is that as long as fencing is configured properly and both
+nodes are available, the cluster should recover without manual intervention.
 
-#### Graceful vs. Unplanned Reboots
+| Section                                                                                 | Description                                                                                               |
+| --------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| [Comparing Failure Modes Across Topologies](#comparing-failure-modes-across-topologies) | A table detailing a high-level overview of the limitations of this topology when compared to others       |
```

Comment:
One of the objections to the table was that it described some "options" more than "topologies". Do we want to keep the term here for better understanding, or shall we use "options" or similar term as well?

Reply:
That's a good point. :) Updated!